File: app/src/main/java/co/smartreceipts/android/purchases/apis/purchases/PurchaseResponse.java
package co.smartreceipts.android.purchases.apis.purchases;

import java.util.Date;

public class PurchaseResponse {

    public MobileAppPurchase mobile_app_purchase;

    public static final class MobileAppPurchase {
        public String id;
        private String user_id;
        private String pay_service;
        private String purchase_id;
        private String status;
    }

}

File: app/src/main/java/com/wops/receiptsgo/purchases/apis/purchases/PurchaseResponse.kt
package com.wops.receiptsgo.purchases.apis.purchases

import java.util.*

data class PurchaseResponse(
    val status: String?,
    val errors: List<String>?,
    val purchase: MobileAppPurchase?
) {

    class MobileAppPurchase constructor(
        val id: String?,
        val user_id: String?,
        val pay_service: String?,
        val purchase_id: String?,
        val product_id: String?,
        val package_name: String?,
        val purchase_time: Date?,
        val subscription_active: Boolean?,
        val created_at: Date?,
        val updated_at: Date?
    )
}